axelor-base/src/test/java/com/axelor/apps/base/test/TestCompanyDateService.java

```diff
@@ -0,0 +1,61 @@
+package com.axelor.apps.base.test;
+
+import com.axelor.apps.base.AxelorException;
+import com.axelor.apps.base.db.Company;
+import com.axelor.apps.base.service.CompanyDateService;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TestCompanyDateService {
+
+  private static final String DD_MM_YYYY_HH_MM = "dd/MM/yyyy HH:mm";
+  private static final String M_D_YYYY_H_MM_A = "M/d/yyyy, h:mm a";
+  private static final String DD_MM_YYYY = "dd/MM/yyyy";
+  private static final String M_D_YYYY = "M/d/yyyy";
+
+  protected CompanyDateService companyDateService;
+  protected Company usCompany;
+  protected Company frCompany;
+
+  @Before
+  public void prepare() {
+    companyDateService = new CompanyDateService();
+    usCompany = prepareCompany(Locale.US.toLanguageTag());
+    frCompany = prepareCompany(Locale.FRANCE.toLanguageTag());
+  }
+
+  @Test
+  public void testDateFormatter() throws AxelorException {
+    Assert.assertEquals(getDateTimeFormat(M_D_YYYY), getCompanyDateformat(usCompany));
+    Assert.assertEquals(getDateTimeFormat(DD_MM_YYYY), getCompanyDateformat(frCompany));
+  }
```

Comment:
please split so we can have one assert by method (for example here: `testDateFormatterUs()` `testDateFormatterFr()`) as this is a good practice to make one assert per method.

Please do the same for the other method